clox/params.py

```diff
@@ -58,8 +57,24 @@
     25: 'epic',
 }
 
+DATE_FORMATS_MAP = {
+    'ISO': '%Y-%m-%d',
+    'US': '%m/%d/%Y',
+    'US-SHORT': '%m/%d/%y',
+    'EU': '%d/%m/%Y',
+    'EU-SHORT': '%d/%m/%y',
+    'DOT': '%d.%m.%Y',
+    'DASH': '%d-%m-%Y',
+    'YMD': '%Y/%m/%d',
+    'DMY': '%d %b %Y',
+    'MDY': '%B %d, %Y',
```

Comment:
these three are better to be consistent. if we're using `%m` somewhere and `%B` and `%b` other places we should change the names.

Reply:
fixed in https://github.com/sepandhaghighi/clox/pull/63/commits/a8f19b9ef8dd761e2c388aaf2715da88d2edd0b4